keymanapp/keyman
# chore: Xcode 14.2 update

When moving from Xcode 13.4 to 14.2, Keyman for Mac builds successfully, but this is not the case for Keyman for iOS.
Two changes generated warnings and caused the build to fail. These changes are described in the Xcode 14 [release notes](https://developer.apple.com/documentation/Xcode-Release-Notes/xcode-14-release-notes):

1) increase the iOS target version from 9 to 11

>The Xcode 14 release supports on-device debugging in iOS 11 and later, tvOS 11 and later, and watchOS 4 and later. Xcode 14 requires a Mac running macOS Monterey 12.5 or later.

2) disable bit code

The xcode 14 release notes describe the deprecation of bit code:

>Deprecations
Starting with Xcode 14, bitcode is no longer required for watchOS and tvOS applications, and the App Store no longer accepts bitcode submissions from Xcode 14.
Xcode no longer builds bitcode by default and generates a warning message if a project explicitly enables bitcode: “Building with bitcode is deprecated. Please update your project and/or target settings to disable bitcode. The capability to build with bitcode will be removed in a future Xcode release. IPAs that contain bitcode will have the bitcode stripped before being submitted to the App Store. Debug symbols for past bitcode submissions remain available for download. (86118779)

With the change of the iOS target version to 11, some code written for older versions of iOS could be removed from  KeyboardCalibrator.

Heads up to @chrisvire @rpmargetts @davidmoore1 that we are making these changes for Keyman for iOS. This PR is in place of #7433.

## User Testing
A basic smoke test is needed for Keyman for macOS and Keyman for iOS

* **TEST_MAC** 
1. Verify that Keyman for macOS is operating as expected.

* **TEST_IOS**
1. Verify that Keyman for iOS is operating as expected.
